### PR TITLE
Added Catppuccin theme

### PR DESCRIPTION
**Pull request by tsjazil**
Soothing Pastel theme

### CODE REVIEW

**Comment by Bruception** [comment]
Please attach screenshots of the theme.

**Comment by tsjazil** [comment]
> Please attach screenshots of the theme.

catppuccinmonkeytype

**Comment by ryujinscales** [comment]
add colorful mode and flip test colors screenshot

**Comment by tsjazil** [comment]
> add colorful mode and flip test colors screenshot

fliptest-catppuccin

**Comment by Miodec** [comment]
You need to add the theme to the `_list` file

**Comment by tsjazil** [comment]
Added

**Comment by Miodec** [comment]
Also i think the name should be changed. Sounds too much like something else.

**Comment by tsjazil** [comment]
> Also i think the name should be changed. Sounds too much like something else.

https://github.com/catppuccin/catppuccin

Its the official name of that colorscheme that's why i used it

File: static/themes/_list.json [hunk 1]
```diff
@@ -719,4 +719,11 @@
     "bgColor": "#dadbdc",
     "mainColor": "#3846b1"
 	}
+  ,{
+    "name": "catppuccin",
+    "bgColor": "#1e1e2e",
+    "mainColor": "#abe9b3"
+	}
 ]
+
```
Comment (Bruception) [inline static/themes/_list.json:728]: Remove these trailing new lines.

Comment (tsjazil) [reply static/themes/_list.json]: 👍🏽

**Comment by Miodec** [comment]
> > Also i think the name should be changed. Sounds too much like something else.
> 
> https://github.com/catppuccin/catppuccin
> 
> Its the official name of that colorscheme that's why i used it

Oh, i see than.

**Review by Miodec — APPROVED** [review]

**Comment by Miodec** [comment]
Thanks

**Pull request merged**